.octopus/deployment_process.ocl

```diff
@@ -119,18 +112,20 @@ step "create-pull-request-to-update-formula-in-homebrew" {
                     -replace "macos_arm_sha256 = `".*`"", "macos_arm_sha256 = `"$macos_arm_sha256`"" `
                     -replace "linux_intel_sha256 = `".*`"", "linux_intel_sha256 = `"$linux_intel_sha256`"" `
                     -replace "linux_arm_sha256 = `".*`"", "linux_arm_sha256 = `"$linux_arm_sha256`"") | Set-Content $formulaFile
-                
-                if ($OctopusParameters) {
-                    git config user.name $OctopusParameters["Publish:HomeBrew:Username"]
-                    git config user.email $OctopusParameters["Publish:HomeBrew:UserEmail"]
-                    $env:GH_TOKEN = $githubtoken
-                }
+
                 git commit -a -m "octopus-cli $packageVersion"
-                
                 git push --set-upstream origin $branchName
-                
-                &($gh) pr create --head $branchName --title "octopus-cli $packageVersion" --body "automatically generated"
-                
+
+                if ($githubtoken) {
+                    $baseBranch = $(git symbolic-ref "refs/remotes/origin/HEAD") -replace "refs/remotes/origin/", ""
+
+                    curl -X POST `
```

Comment:
Does this work? We're inside powershell, which sets an alias for `curl` to equal `Invoke-WebRequest` which is dumb and annoying as almost all the flags aren't the same as curl. Github actions runners do have the win32 versions of bash, curl, etc installed though, so if it lands on that then it'll be ok.

IMHO it'd be cleaner if we just had powershell do the web request natively, and not take any dependencies on linux stuff, it looks like this is pretty trivial
https://www.thecodebuzz.com/making-powershell-post-request-with-body-json-array/

```
Invoke-WebRequest -Method "POST" -Uri https://api.github.com/repos/OctopusDeploy/homebrew-taps/pulls `
-Body.. etc
```

Reply:
Honestly forgot we were in PowerShell here, so just straight copied the code from the GitHub API docs. Will update to make sure it works in PowerShell

Reply:
I've converted it to use `Invoke-RestMethod` instead of `curl`